packages/suite/src/middlewares/suite/sentryMiddleware.ts

```diff
@@ -122,6 +123,10 @@ const sentryMiddleware =
                 });
                 break;
             }
+            case deviceAuthenticityActions.result.type:
+                if (action.payload.result.valid) return;
+                setSentryContext('device-authenticity', action.payload.result);
```

Comment:
TODO: this is not reporting to sentry, just adding the context for future errors...